functions/clean_blank_lines.pp

```diff
@@ -0,0 +1,8 @@
+# @summary Remove blank lines from a string
+# @param content The content to remove blank lines from
+# @return The content with blank lines removed
+function unbound::clean_blank_lines (
```

Comment:
I think this is generic enough that it could also be part of extlib?

Reply:
https://github.com/voxpupuli/puppet-extlib/pull/239